code/modules/power/apc.dm

```diff
@@ -1448,7 +1448,7 @@ var/zapLimiter = 0
 
 			switch(lowertext(data["command"]))
 				if ("status")
-					src.post_status(src.host_id,"command","term_message","data","command=status&charge=[cell ? round(cell.percent()) : "00"]&equip=[equipment]&light=[lighting]&environ=[environ]&cover=[coverlocked]")
+					src.post_status(src.host_id,"command","term_message","data","command=status&area=[ckey("[src.area]")]&charge=[cell ? round(cell.percent()) : "00"]&equip=[equipment]&light=[lighting]&environ=[environ]&cover=[coverlocked]")
```

Comment:
why in the holy fuck does this and all other computer code use the ckey proc, which is for finding the canonical form of player's keys

what the fuck

Reply:
That is an excellent question, but I am a dumb code baby and can't answer that one

Reply:
ok after a lot of looking, aibm didn't want to roll an in-house sanitize (completely fair) for this, because ckey will sanitize any string thrown at it

someone else can refactor this, out of scope